App/Screens/About/About.tsx

```diff
@@ -27,12 +27,11 @@ import {
 import { ScrollIntoView, wrapScrollView } from 'react-native-scroll-into-view';
 import { scale } from 'react-native-size-matters';
 import { NavigationInjectedProps } from 'react-navigation';
-
-import { Box } from './Box';
 import { BackButton } from '../../components';
 import { i18n } from '../../localization';
 import { trackScreen } from '../../util/amplitude';
 import * as theme from '../../util/theme';
+import { Box } from './Box';
```

Comment:
I like this rule! putting deeper imports first. Is it some semi-official rule, or your custom?

Reply:
I like too, but is part of my custom settings :)